scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+
+ENV JAVA_HOME=/usr/lib/jvm/java-11.0.15-openjdk
+ENV PATH="$JAVA_HOME/bin:${PATH}"
+
+# Install sbt user qa
+RUN curl -L -o /home/qa/sbt.zip https://github.com/sbt/sbt/releases/download/v1.5.7/sbt-1.5.7.zip \
+	&& unzip /home/qa/sbt.zip -d /home/qa \
+	&& rm /home/qa/sbt.zip
+
+# Put tools like aws and sbt in the PATH
+ENV PATH /home/qa/.local/bin:/home/qa/sbt/bin:/home/qa/bin:${PATH}
+RUN sudo ln -s /home/qa/sbt/bin/sbt /usr/local/bin/sbt
```

Hadolint

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)